.github/workflows/test-package-build.yml

```diff
@@ -52,16 +49,21 @@ jobs:
         run: |
           echo "MLFLOW_SKINNY=true" >> $GITHUB_ENV
 
-      - name: Build UI
-        if: ${{ matrix.type == 'full' }}
-        working-directory: mlflow/server/js
-        env:
-          # Prevent warnings (emitted from react-pdf) from being treated as errors
-          # https://github.com/wojtekmaj/react-pdf/issues/280
-          CI: false
+      - name: Create dummy index.html
```

Comment:
This jobs finished within a minute:

https://github.com/mlflow/mlflow/runs/6058951767?check_suite_focus=true

Reply:
The latest run of this job on master took 7.5 min:

https://github.com/mlflow/mlflow/runs/6058416663?check_suite_focus=true

Reply:
It's 6.5 min speed up.